### DISCUSSION TITLE
is LMMS dead?

### DISCUSSION THREAD

**Discussion by CaBachhav** [LMMS/lmms#7632] [General]
is lmms a dead project? its 2 years or so it is not being updated. What is going wrong with the developer team members. Lmms community have rights to know it

**Comment by qnebra** [comment]
Indeed, it is dead project, with new pull requests being merged nearly weekly, sometimes daily. While I do not know why there is not yet new alpha / test version for 1.3 being released, project itself, at this moment, is far from dead.

Also remember, lmms team is entirely volunteer based, with each team member working when they can.

**Comment by headquarter8302** [comment]
If you want to have the bleeding edge builds, go here: https://github.com/LMMS/lmms/actions/workflows/build.yml, go to the most recent non-failing build (the one that's ticked not crossed), and download the build artifact that's compatible with your environment (`msvc` or `mingw` for Windows, `linux` for Linux, `macos-arm64` for M-series devices and `macos-x86_64` for Intel-series devices).

The project is not dead, it just doesn't publish interim releases as much
